README.md

```diff
@@ -13,81 +13,25 @@ displays the contacts themselves, fetching them from content store:
 
 ## Technical documentation
 
-This is a Ruby on Rails application that provides a private admin UI for signon users with permission to allow them to manage the contacts for their organisation.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-The app has its own database (MySQL), which is used by the admin UI. This is in contrast to the separate [government-frontend](https://github.com/alphagov/government-frontend) app which reads contacts from the [content-store](https://github.com/alphagov/content-store). The admin UI part of this app handles publishing the contacts to the [publishing-api](https://github.com/alphagov/publishing-api) so that they are present for [finder-frontend](https://github.com/alphagov/finder-frontend) and [government-frontend](https://github.com/alphagov/government-frontend) to read.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-### Dependencies
+**Use GOV.UK Docker to run any commands that follow.**
 
-* Ruby 2.3.0
-* MySQL
-* [whitehall](https://github.com/alphagov/whitehall)
-  1. to access the organisations API and maintain parity with the organisations managed in whitehall.
-  2. to access the world locations API to provide the correct list of countries for the country dropdown in the address form
```

Comment:
Whilst I agree we can remove the `Dependencies` section, it might be nice to leave some of this useful context:

```suggestion
Contacts Admin uses Whitehall to access the organisations API (maintaining parity with the organisations managed in Whitehall) and to access the world locations API (to provide the correct list of countries for the country dropdown in the address form).
```

Without it, the developer has to look in the code to discover the (surprising) relationship with Whitehall. I think integrations with Signon and Content Store are pretty standard, but the Whitehall dependency is a snowflake that it can't hurt to document, IMO.

Reply:
Fair enough. I've amended the first commit with tweaked wording. I think this goes better in the "before running the app" section, as it's only relevant there.